packages/builder/src/pages/builder/index.svelte

```diff
@@ -5,11 +5,9 @@
   auth.checkQueryString()
 
   $: {
-    if (!$auth.user) {
-      $redirect(`./auth`)
-    } else if ($auth.user.builder?.global) {
+    if ($auth.user?.builder?.global) {
```

Comment:
was the `(!$auth.user)` redundant here?

Reply:
specifically around the redirect
```
$redirect(`./auth`)
```

is that taken care of somewhere else now?

Reply:
re-read the description, all good here 👍

Reply:
Yeah, there was already somewhere that it caught the login, with much better rules and checks around it to stop loops, specifically:
```js
    // Redirect to log in at any time if the user isn't authenticated
    else if (
      loaded &&
      (hasAdminUser || cloud) &&
      !$auth.user &&
      !$isActive("./auth") &&
      !$isActive("./invite") &&
      !$isActive("./admin")
    ) {
      const returnUrl = encodeURIComponent(window.location.pathname)
      $redirect("./auth?", { returnUrl })
    }
```
in the `pages/builder/_layout.svelte`